pkg/registry/core/pod/storage/eviction.go

```diff
@@ -211,9 +211,10 @@ func (r *EvictionREST) Create(ctx context.Context, name string, obj runtime.Obje
 		}
 
 		if len(pdbs) > 1 {
+			msg := fmt.Sprintf("This pod has %d PodDisruptionBudgets (%s, %s  ...) and the eviction subresource supports 1.", len(pdbs), pdbs[0].Name, pdbs[1].Name)
```

Comment:
The way we currently list pods here is we don't look at ownership nor RBAC, so by just printing random 2 pods you risk that you'll expose information that is not meant for the user seeing that information.